### ISSUE TITLE
New ordonate axe

### ISSUE THREAD

**Issue by Julien2313** [amochtar/aoc-ranking#1]
Hi, I was thinking about improving your representation of the graph.
At the moment, you're only showing the rank, without taking into account the difference of score.

Can't we imagine, in example, calculating for each day the theoretical maximum of stars (it would be 100%), and divide the score by the maximum. So, we would keep the rank, but add the difference of score.

Example (cause I might(certainly) not be clear):
Day 1, 3 players, maximum 6 points
P1 5 => 0.83 1st
P2 4 => 0.66 2nd
P3 3 => 0.50 3rd

Day 2, maximum 12 points
P1 5+6 => 0.92 1st
P2 4+2 => 0.50 3rd
P3 3+4 => 0.58 2nd

Let me know if you don't understand something 👍

**Comment by amochtar** [comment]
Hi @Julien2313, sorry for the late reply. Since a new round of AoC is starting soon I was only just now revisiting this project :smile: 

I did try something like you suggested, but I found that it made the chart far less readable, since some lines will be a lot closer to each other. I am thinking about adding the actual scores, so you have some insight into how close people are together. I'm not yet sure how though...
